file_status_extended.go

```diff
@@ -0,0 +1,28 @@
+package smartling
+
+// FileStatusExtended describes file translation status obtained by GetFileStatusExtended
+// method.
+type FileStatusExtended struct {
+	File
+
+	AuthorizedStringCount int
+	AuthorizedWordCount   int
+	CompletedStringCount  int
+	CompletedWordCount    int
+	ExcludedStringCount   int
+	ExcludedWordCount     int
+	TotalStringCount      int
+	TotalWordCount        int
+}
+
+func (fs FileStatusExtended) AwaitingAuthorizationStringCount() int {
+	return fs.TotalStringCount - fs.AuthorizedStringCount - fs.ExcludedStringCount - fs.CompletedStringCount
+}
+
+func (fs FileStatusExtended) InProgressStringCount() int {
+	return fs.NotCompletedStringCount() - fs.AwaitingAuthorizationStringCount()
```

Comment:
`fs.AuthorizedStringCount` this is what translator sees. So all authorized strings are "in progress" strings